OpenRA.Utility/Program.cs

```diff
@@ -28,7 +28,6 @@ static void Main(string[] args)
 			AppDomain.CurrentDomain.AssemblyResolve += GlobalFileSystem.ResolveAssembly;
 
 			Log.AddChannel("perf", null);
-			Log.AddChannel("debug", null);
```

Comment:
The utility is not supposed to use that channel anyway.
Looks to me like you are removing this from here so you can add in in the replay parsing because you copied that part from somewhere else, which is another issue that @pchote's comment below is talking about.